### PR TITLE
Feature bp 736 repalce movai classes with backend repo

### PR DESCRIPTION
**Pull request by erezz-mov-ai**
necessary changes to split movai_classes
jira:
https://movai.atlassian.net/browse/BP-736?atlOrigin=eyJpIjoiZjk3NGVmMDkyYmU5NDUxNGJhMjQ4YzhjY2IwNmNmNmEiLCJwIjoiaiJ9
- [x] Make sure you are opening from a **topic/feature/bugfix branch**
- [x] Ensure that the PR title represents the desired changes
- [x] Ensure that the PR description detail the desired changes
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

[^note]:
    Put an `x` into the [ ] to show you have filled the information.
    The template comes from https://github.com/MOV-AI/.github/blob/master/.github/pull_request_template.md
    You can override it by creating .github/pull_request_template.md  in your own repository

### CODE REVIEW

File: backend/core/README.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Mov.ai Core Module
+
```
Comment (erezz-mov-ai) [inline backend/core/README.md:2]: move the contents to dal.plugins

File: backend/core/permission.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
+"""
+   Copyright (C) Mov.ai  - All Rights Reserved
+   Unauthorized copying of this file, via any medium is strictly prohibited
+   Proprietary and confidential
+
```
Comment (erezz-mov-ai) [inline backend/core/permission.py:5]: add comment in the file docstring: Work in progress: Still in development

File: backend/core/plugin.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
+"""
```
Comment (erezz-mov-ai) [inline backend/core/plugin.py:1]: remove the file, it moved to dal.plugins

File: backend/tools/new_user.py [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # create a new user
 import argparse
-from dal.models import User
-from dal.scopes import scopes
+from dal.scopes.scopestree import scopes
 
+from .models.user import User
```
Comment (erezz-mov-ai) [inline backend/tools/new_user.py:5]: change to dal.models.user
change to dal.models.scopestree

File: runtime-requirements.txt [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# runtime requirements.
```
Comment (erezz-mov-ai) [inline runtime-requirements.txt:1]: stay only with build-requirements.txt

File: backend/core/README.md (hunk 1 above)
Comment (erezz-mov-ai) [reply backend/core/README.md:2]: done

File: backend/core/permission.py (hunk 2 above)
Comment (erezz-mov-ai) [reply backend/core/permission.py:5]: done

File: backend/core/plugin.py (hunk 3 above)
Comment (erezz-mov-ai) [reply backend/core/plugin.py:1]: done

File: backend/tools/new_user.py (hunk 4 above)
Comment (erezz-mov-ai) [reply backend/tools/new_user.py:5]: done

File: runtime-requirements.txt (hunk 5 above)
Comment (erezz-mov-ai) [reply runtime-requirements.txt]: done

**Review by Mograbi — APPROVED** [review]

**Pull request merged**